sympy/solvers/tests/test_solveset.py

```diff
@@ -2313,3 +2313,7 @@ def test_solve_modular_fail():
             ImageSet(Lambda(n, 74*n + 31), S.Integers)
 
 # end of modular tests
+
+def test_issue_17276():
+    assert nonlinsolve([Eq(x,5**1/5), Eq(x*y, 25*sqrt(5))], x, y) == \
```

Comment:
Is there a typo? The first equation `Eq(x, 1.0)` leaves nothing to be solved.
```
>>> 5**1/5
1.0
>>>
```

Reply:
oops sorry it went unnoticed

Reply:
> why it is giving assertion even though `nonlinsolve([Eq(x,5**(Integer(1)/5)), Eq(x*y, 25*sqrt(5))], x, y)` gives the same output.
> 
> ```
> assert nonlinsolve([Eq(x,5**(Integer(1)/5)), Eq(x*y, 25*sqrt(5))], x, y) == FiniteSet((5**(1/5), 25*5**(3/10)))
> ```

What should I do here?

Reply:
Ping @jksuom

Reply:
The equations are now trivial. It would probably suffice to sympify some numbers.
```suggestion
    assert nonlinsolve([Eq(x,5**(S(1)/5)), Eq(x*y, 25*sqrt(5))], x, y) == \
                FiniteSet((5**(S(1)/5), 25*5**(S(3)/10)))
```

Reply:
Actually I was talking about the below example as it was mentioned in the issue.
when I add a test for them it gives assertion error. I don't know why.
```
>>> nonlinsolve([Eq(x, 5**(Integer(1)/5)), Eq(x*y, 25*sqrt(5))], x, y)
FiniteSet((5**(1/5), 25*5**(3/10)))
```
This is the test I have added:
```
def test_issue_17276():
    assert nonlinsolve([Eq(x, 5**(Integer(1)/5)), Eq(x*y, 25*sqrt(5))], x, y) == \
    FiniteSet(5**(1/5),25*5**(3/10))
```

Reply:
The assertion fails because there are Floats on the right hand side:
```
In [2]: print(nonlinsolve([Eq(x, 5**(Integer(1)/5)), Eq(x*y, 25*sqrt(5))], x, y))
FiniteSet((5**(1/5), 25*5**(3/10)))

In [3]: FiniteSet((5**(1/5), 25*5**(3/10)))
Out[3]: {(1.37972966146121, 40.5164149173191)}
```

Reply:
very sorry for late reply (Actually My universities exams are going on now.)

Is there any way using which I can stop this expression (`FiniteSet((5**(1/5), 25*5**(3/10)))`) from getting simplified to Float?
I know how to simplify the expression but it seems exactly reverse.

Reply:
The numbers should be sympified by adding `S`, for example. See [this suggestion](https://github.com/sympy/sympy/pull/18670#discussion_r383307449) (the second line).